assets/css/colors.scss

```diff
@@ -0,0 +1,10 @@
+$yellow: #FDC129;
+$light-turquoise: #EBF8F2;
+$apricot: #fbf2ea;
+$pink: #efb7b6;
+$brand-info: #4572C0;
+$brand-info-light: #e8eff9;
+$brand-success-light: #f2f7eb;
+$turquoise: #31A3AE;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#31A3AE" to be "#31a3ae" (color-hex-case)](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5681062)